components/checkbox.module.css

```diff
@@ -0,0 +1,22 @@
+.checkbox {
+  position: relative;
+  height: 24px;
+  width: 24px;
+  border: 1px solid #d1d6dc;
+  border-radius: 4px;
+
+  &:after {
+    content: "";
+    position: absolute;
+    top: -5px;
+    left: -5px;
+    height: calc(100% + 10px);
+    width: calc(100% + 10px);
+    border: 2px solid #dc2626;
+    border-radius: 8px;
+  }
+  &[aria-checked="true"] {
```

Comment:
windi doesn’t have a variant for this nor is there a plugin, so we should keep this ruleset here for now, but since i did a little looking into it:

it turns out their plugin api [makes it pretty easy](https://windicss.org/plugins/interfaces.html#add-variants) to add a variant.

i also put in an issue proposing that aria attributes could be supported in the core variants so e.g. we could use `checked:*` here: https://github.com/windicss/windicss/issues/555